### DISCUSSION TITLE
User icon

### DISCUSSION THREAD

**Discussion by DaanKorver** [JustinLung/the-game-of-tones-chapterpage#13] [Questions] [answered]
We missen het user icon, zou deze nog geexporteerd kunnen worden?
![Screenshot from 2021-12-13 12-36-19](https://user-images.githubusercontent.com/30351629/145805447-a86eba80-d390-4e27-a676-ba9ef34ef327.png)

**Comment by JustinLung** [comment, marked as answer]
We can use another icon, it doesn't matter
